manifest.md

```diff
@@ -84,14 +84,22 @@ A client will distinguish a manifest list from an image manifest based on the Co
 - **`annotations`** *string-string hashmap*
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
+    Annotations MUST BE a key-value, unordered hashmap where both the key and value MUST be strings.
```

Comment:
Can we avoid duplicate entries for manifest and manifest-list?  The manifest is more fundamental, so I'd rather replace this list entry with:

    - **`annotations`** (string → string map, optional) as defined for [manifest](#image-manifest).